cardigan/stories/components/BetaMessage.js

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from '@storybook/react';
+import BetaMessage from '../../../common/views/components/BetaMessage/BetaMessage';
+import Readme from '../../../common/views/components/BetaMessage/README.md';
+import { ThemeProvider } from 'styled-components';
+import theme from '../../../common/views/themes/default';
+
+const stories = storiesOf('Components', module);
+
+stories.add(
+  'BetaMessage',
+  () => (
+    <ThemeProvider theme={theme}>
```

Comment:
Maybe we should wrap the whole story book in the theme?

Reply:
That sounds like a good shout.